packages/docs/content/components/uSA-banner.mdx

```diff
@@ -10,8 +10,7 @@ healthcare:
   sketchLink: 25qAbzl
 ---
 
-
```

Comment:
The name of this file has a lowercase `u` followed by `SA` in caps, let's update to make consistent

Reply:
I was afraid to because we ran into some big git issues another time we tried to fix casing on a file. I'll attempt it 🤞

Reply:
No worries if not, thanks!